prov/gni/include/gnix_bitmap.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2015 Cray Inc. All rights reserved.
+ * Copyright (c) 2015-2017 Cray Inc. All rights reserved.
```

Comment:
If we didn't modify this file last year we shouldn't be using the -, it should probably just be Copyright 2015, 2017.